package.json

```diff
@@ -13,8 +13,9 @@
   "scripts": {
     "dev-up": "npm install && (cd app && npm install) && npm run build",
     "test": "gulp test",
+    "test-watch": "mocha --watch --require babel-register --recursive ./test",
     "lint": "eslint .",
-    "ci": "gulp test && npm run lint",
+    "ci": "gulp build test && npm run lint",
```

Comment:
This I missed and it should actually be now calling `gulp tdd`. I'll push a change now and you can see.